methods/did:.md

```diff
@@ -0,0 +1,33 @@
+# did:*
+
+The "any" DID method encoding allows any DID to be represented as a multidid by essentially putting the entire DID string in the URL portion of the multidid byte string.
+
+## Method multicodec
+
+`0x00`
+
+## Format
+
+```
+<multicodec-did><method-multicodec><method-bytes><url-length><url-bytes>
+```
+
+Where:
+
+* `multicodec-did` - the value `0xd1d`
+* `method-multicodec` - a varint encoding of `0x00`
+* `method-bytes` - MUST contain exactly zero bytes
+* `url-length` - a varint describing the length of the `url-bytes` parameter
+* `url-bytes` - a UTF-8 encoded string representing the DID and URL component (e.g. everything except `did:`)
```

Comment:
I could be wrong, but: URI not URL, right?

I'm all for i18n, but do DIDs support [IRI](https://en.wikipedia.org/wiki/Internationalized_Resource_Identifier), or would these need to be ASCII instead of UTF8? The [DID spec](https://www.w3.org/TR/did-core/) has a bunch of search hits for ASCII, but none for UTF8.